server/src/main/java/org/opentosca/toscana/plugins/cloudformation/handler/OperationHandler.java

```diff
@@ -80,7 +96,7 @@ public void handleStart(RootNode node, String computeHostName) {
      @param filePath   path to the artifact
      @param serverName name of the Instance
      */
-    public void addCreate(String filePath, String serverName) throws IOException {
+    public void addCreate(String filePath, String serverName) {
         markUtilFile(filePath);
         CFNFile cfnFile = handleOperationFile(filePath, MODE_500, serverName);
```

Comment:
Is there also a TODO to check which mode is needed or is `MODE_500` the right one?